src/api/spec/features/beta/webui/projects_spec.rb

```diff
@@ -15,13 +15,19 @@
     expect(page).to have_css('h3', text: project.title)
   end
 
-  it 'project status' do
-    login user
-    broken_package_with_error
-    visit project_status_path(project_name: project)
-    uncheck('limit_to_fails')
-    click_button('Filter results')
-    expect(page).to have_text('Status of')
+  describe 'Project Status' do
+    before do
+      login user
+      broken_package_with_error
+      visit project_status_path(project_name: project)
+      uncheck('limit_to_fails')
+      click_button('Filter results')
+    end
+
+    it "dislays the expected packages in the results' table" do
+      expect(page).to have_text('Displaying 1 packages that need handling')
```

Comment:
These 2 specs are also passing on `master`, so I'm not confident that they are really proving that the issue is fixed now with the changes included in this PR.

Reply:
I think you need a package in build state 'failed' to test for the issue

Reply:
without the failed comment attribute set

Reply:
I'll take the tests out of this PR.